sponsorlytix_selenium/driver.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.ui import WebDriverWait
+
+from config import get_config
+
+
+configs = get_config()
+
+
+class SponsorlytixDriver:
+
+    def __init__(self, crawler_name='Social Media Crawler'):
+        self.crawler_name = crawler_name
+        if configs.driver_remote:
+            self.driver = self.__get_remote_driver()
+        else:
+            self.driver = self.__get_web_driver()
+        self.wait = WebDriverWait(self.driver, configs.timeout)
+
+    def __get_remote_driver(self):
+        driver_host = "3.16.155.212"
+        driver_user = "sponsorlytix"
+        driver_password = "HXMswfgn7E7Y"
```

Comment:
remove this mocks, we need to encrypt in our env variables.

Reply:
Done